docs/build.sh

```diff
@@ -7,6 +7,9 @@
 
 set -e
 
+ENVOY_SRCDIR=$(pwd)
+export ENVOY_SRCDIR
```

Comment:
can you do it with `SRC_DIR` and `realpath`

ie move further down and add after SRC_DIR like this

```

SRC_DIR="$(dirname "$SCRIPT_DIR")"
ENVOY_SRCDIR="$(realpath "$SRC_DIR")"
```

Reply:
that way it will be using the same path as the existing env var that does the ~same thing and also it will work in a bazel target where `.` wont

Reply:
I will do that. Thanks